code/game/turfs/flooring/seasonal.dm

```diff
@@ -0,0 +1,216 @@
+var/season
+
+/proc/do_season()
```

Comment:
Maybe more obvious name for that proc could be in order like get_current_season or check_current_season